### DISCUSSION TITLE
Is there a discord server for fooocus specifically?

### DISCUSSION THREAD

**Discussion by martinenkoEduard** [lllyasviel/Fooocus#1441] [General]
Is there a discord server for fooocus specifically?

**Comment by mashb1t** [comment]
No, there isn't. You can find more information here: https://github.com/lllyasviel/Fooocus/issues/112

**Comment by ItsNoted** [comment]
Not official. But there is also an unofficial [sub Reddit](https://www.reddit.com/r/fooocus/).

**Comment by ReedRyan** [comment]
I want to download Fooocus, but not getting anything from Github, Please help.

**Reply by mashb1t** [reply]
@ReedRyan please find instructions how to install Fooocus in the readme.
